flex-squid/src/main/java/org/sonar/flex/FlexAstScanner.java

```diff
@@ -167,7 +147,7 @@ public SourceCode createSourceCode(SourceCode parentSourceCode, AstNode astNode)
            FlexGrammar.EMPTY_STATEMENT)
         .build());
 
-    builder.withSquidAstVisitor(new ComplexityVisitor());
+    //builder.withSquidAstVisitor(new ComplexityVisitor());
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.flex%3Aflex&branch=SONARFLEX-105&issues=AWGd6bAnR7xclXvFe_Ib&open=AWGd6bAnR7xclXvFe_Ib)

[ISSUE_KEY]: AWGd6bAnR7xclXvFe_Ib